type_templates/vrm.js

```diff
@@ -176,6 +176,20 @@ export default e => {
       };
       _unskin();
 
+      const _addCapsulePhysics = () => {
+        const avatarHeight = 1.4467394369000004;

```

Comment:
Avatar height should be gotten from the avatar? Also this is extremely random...